app/charges/controllers.py

```diff
@@ -155,8 +155,14 @@ def get_charge(user, user_data, broadcast = False):
 @socketio.on('create_charge')
 @ensure_dict
 @get_user
-def create_charge(user, user_data):
+def create_charge(user, user_data):    
     committee = Committees.query.filter_by(id = user_data.get("committee","")).first()
+    
+    # The charge title must be at least 2 characters long and cannot contain special characters
+    invalid_chars = r'/^()#@![]{}`~\?%*:|"<>.'
+    if user_data['title'] is None or len(user_data['title']) <= 1 or any(char in user_data['title'] for char in invalid_chars):
```

Comment:
So this is just a style thing, I have never been a fan of large concatenated if statements. I like to do the individual checks and save them to a variable before checking their truthiness. Here's an example from naps: https://github.com/ritstudentgovernment/laravel-naps/blob/2201f837ff4c6c341fd2ca76caf3f37579fb8d89/app/Http/Controllers/SpotController.php#L123

Reply:
Just fixed it, looks much cleaner now than before i agree.